ens/utils.py

```diff
@@ -103,27 +126,27 @@ def is_valid_name(name):
         return False
 
 
-def to_utc_datetime(timestamp):
+def to_utc_datetime(timestamp: float) -> Optional[datetime.datetime]:
     if timestamp:
         return datetime.datetime.fromtimestamp(timestamp, datetime.timezone.utc)
     else:
         return None
 
 
-def sha3_text(val):
+def sha3_text(val: Any) -> HexBytes:
```

Comment:
Should probably be [`AnyStr`](https://docs.python.org/3/library/typing.html#typing.AnyStr) if you want to include `bytes` types or just `str` if not (but it looks like it supports both)